modules/installation/pages/client-requirements.adoc

```diff
@@ -23,13 +23,15 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
 | {sls}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sls}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sls} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
+| {sls} for SAP 15 | {x86}_64, {ppc}                    | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 8 | {x86}_64                                   | icon:question[role="gray"]    | icon:question[role="gray"]
 | {rhel} 7 | {x86}_64                                   | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 6 | {x86}, {x86}_64                            | icon:check[role="green"]      | icon:check[role="green"]
-| CentOS 7 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| CentOS 6 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
+| CentOS 7*) | {x86}, {x86}_64                          | icon:question[role="gray"]    | icon:question[role="gray"]
+| CentOS 6*) | {x86}, {x86}_64                          | icon:question[role="gray"]    | icon:question[role="gray"]
```

Comment:
If it is RES, he is supported (green). This line was meant for plain CentOS.

Reply:
@joesusecom we please need your feedback.

Reply:
Oh, i just see another problem. Expanded Support for version 7 is only available for x86_x64.
If we mean here just plain CentOS 7 the arch list might be correct. Together with Expanded Support it is wrong.